private/pkg/transport/http/httpserver/zap.go

```diff
@@ -90,3 +119,31 @@ func getRequestFields(request *http.Request) []zap.Field {
 		zap.String("path", request.RequestURI),
 	}
 }
+
+// this gets request fields that are relevant but not covered by http request log
```

Comment:
It's convention in go for comments to start with a capitalized first letter.

Additionally, even though this isn't exported, I would still follow the godocs comment convention and start the comment with the function name:

```go
// getTopLevelRequestFields returns zap Fields that are relevant to a request, but not are not a part of httpRequestLog.
func getRequestFields(request *http.Request) []zap.Field {
    return []zap.Field{
		zap.String("host", request.Host),
		zap.String("path", request.RequestURI),
	}
}
```

Reply:
I removed this function because it's no longer needed after I simplified `Write`, as per your comment